javascript/javascript2/week1/warmUp.js

```diff
@@ -0,0 +1,44 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
```

Comment:
which homework is this 😁 ? can't find it in JS2 Week1 did this change come by mistake?

Reply:
This is warmup for JS2 Week1 :)